staff.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Staff
+---
+
+## Identifying Staff
+
+The _staff_ are the volunteers who act as network and system administrators to
+keep Libera.Chat running smoothly.
+
+The complete current list of staff is available using the IRC command
+`/who libera/staff/*`.
+
+When this page was last updated, the staff members were:
+
+staff|staff|staff|staff
+--|--|--|--
+amdj     | emerson   | JonathanD  | nhandler
+bigpresh | emilsp    | jose       | nicole
+D[\_]     | Freso     | kline      | niko
+deedra   | Fuchs     | lausefuchs | njan
+edk      | glguy     | li         | Stx
+el       | jess      | mniip      | Swant
+
+Among the ops in `#libera` is also a service bot (which is not currently a
+member of staff):
+
+| bot
+|-
+| ozone
+
+## Contacting Staff
+
+### On IRC
+
+Assuming you can connect to IRC (`ircs://irc.libera.chat:6697`), that should be
+your first point of contact with staff.
+
+The quickest way to find the list of on-line staff is to look for chanops among
+the members in the `#libera` channel, and to find the complete list to
+
+Use the IRC command `/stats p` to get a list of available staff, and use
+`/msg`_`nick`_ to chat with one of them.
+
+If `/stats p` does not report any online staff, or you can't get a response,
+try asking in the `#libera` channel. If your question is sensitive or private,
+feel free to `/msg` any of the ops in that channel.
+
+### By Email
+
+Generally don't send email until directed to do so by staff, and then they will
+tell you which email address to use.
+
+But if you cannot get on-line with IRC, please use the email address that
+matches your enquiry:
+
+* policy@libera.chat for anything related to the privacy policy
```

Comment:
This is missing support@, the emails also are not links.

However I don't fully agree that listing them all in one place is a great idea, imo we should instead list the relevant email address in the relevant places. Specifically: this doesn't tell you WHAT to email projects@ which will add more work and back and forth to figure out. It doesn't tell you WHAT to email bans@ (arguably https://libera.chat/policies#network-bans should do that better) perhaps the problem is rather that its difficult to find information about a specific thing (and in general I hope we can make some improvements in the guides there long term)

Reply:
I gathered the email addresses from other pages; support@ wasn't included anywhere when I did that. However I'm happy to reduce this to something along the lines of:

    Before sending email, please chat with staff on IRC
    and we will tell you which email address to use.
    If you can't connect to IRC, please read [help with connecting](/someotherpage)

What should I put for _someotherpage_ ?